metadata-ws/src/test/resources/features/analysis.feature

```diff
@@ -0,0 +1,394 @@
+Feature: analysis object
+
+  Scenario: register an analysis successfully
+    When I request POST /reference-sequences with JSON payload:
```

Comment:
i guess the dependent object creations can be rephrased using "**Given** I had created reference-sequences with JSON payload ... And set url to TEST_REF_SEQ "

Reply:
Ok, in the entire code When-Then is used from beginning. If everybody agrees i can change everywhere.